app/uk/gov/hmrc/ngrdashboardfrontend/config/AppConfig.scala

```diff
@@ -16,19 +16,26 @@
 
 package uk.gov.hmrc.ngrdashboardfrontend.config
 
-import javax.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.ngrdashboardfrontend.controllers.routes
+
+import javax.inject.{Inject, Singleton}
 
 
 @Singleton
 class AppConfig @Inject()(config: Configuration) {
   lazy val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
 
-  lazy val feedbackFrontendHost = config.getOptional[String]("microservice.services.feedback-survey-frontend.host").getOrElse(throw new Exception(s"Missing key: microservice.services.feedback-survey-frontend.host"))
+  private lazy val feedbackFrontendHost = config.getOptional[String]("microservice.services.feedback-survey-frontend.host")
+    .getOrElse(throw new Exception(s"Missing key: microservice.services.feedback-survey-frontend.host"))
 
-  lazy val feedbackFrontendUrl = s"$feedbackFrontendHost/feedback/NGR-Dashboard"
+  private lazy val dashboardHost = config.getOptional[String]("dashboard.host")
+    .getOrElse(throw new Exception(s"Missing key: dashboard.host"))
 
-  lazy val basGatewayHost = config.getOptional[String]("microservice.services.bas-gateway-frontend.host").getOrElse(throw new Exception(s"Missing key: microservice.services.bas-gateway-frontend.host"))
+  private lazy val basGatewayHost = config.getOptional[String]("microservice.services.bas-gateway-frontend.host")
```

Comment:
Could we not have a generic function where you pass the key into it rather than duplicating the same piece of code where the key is the only different thing. Also realistically, I don't think we really need this as we will always have the key in application.conf as well so the getOrElse will never be hit. The only error we are likely to see will be in the environments if its not been overridden correctly which you have sorted out in the other PR's